packages/examples/src/quickstart/smart-clock.ts

```diff
@@ -0,0 +1,96 @@
+/********************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the W3C Software Notice and
+ * Document License (2015-05-13) which is available at
+ * https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR W3C-20150513
+ ********************************************************************************/
+
+// This is an example Thing which is a smart clock that runs 60 times faster than real time, where 1 hour happens in 1 minute.
+
+import { Servient } from "@node-wot/core";
+// import { HttpServer } from "@node-wot/binding-http";
```

Comment:
remove? or have the same commented line for 

servient.addServer(new HttpServer(...));